dist/angular-validation.js

```diff
@@ -445,6 +450,18 @@
 
 
                 /**
+                * It is emitted inside $asyncValidators´ directives to force the validation execution
+                */
+                $rootScope.$on('execute-validation', function(event, args) { 
```

Comment:
Inside the $asyncValidators directive as follow:

```
return $http.get('/emailexists', {'data': modelValue})
                        .then(function resolved(result) {
                            console.log('resolved, result: ' + result.data);

                            if (result.data) {
                                $rootScope.$broadcast('execute-validation',
                                        {
                                            $scope: $scope,
                                            element: element,
                                            attrs: attrs,
                                            ngModel: ngModel
                                        });
                                return $q.reject('exists');
                            }

                            return true;

                        }, function rejected() {
                            console.log('rejected');

                            return true;
                            //retorna "true" pois algum erro ocorreu e a validação
                            //não foi realizada corretamente, mas será no server side.
                        });
```

Give a look in the plunker, please, in demo-asyncvalidation.js:
http://plnkr.co/edit/NF1NmW?p=preview